<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>测试选项</string>
   </property>
   <widget class="QPushButton" name="PBT_TEST_COM_CN8_115200">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试115200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CN8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CN8串口测试：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_use_com_tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>现可用端口：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CBBOX_Port">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_SEARCH">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测可用串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_RES">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查看结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_XML">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成XML报告</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_TEST_COM_CN8_38400">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试38400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_TEST_COM_CN8_9600">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试9600</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TIME">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>日期测试：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_TEST_TIME">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_LAN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LAN接口测试：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_LAN_NTE">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试网络功能</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextE_RES_TIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>441</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>测试结果与选项</string>
   </property>
   <widget class="QPushButton" name="PBT_close">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_CN8_tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CN8测试结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_115200">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>待定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_XML">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>XML文件尚未生成</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_SWITCH">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>文件夹转换情况</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_115200tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>115200波特率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38400tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>148</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>38400波特率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_38400">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>待定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9600tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>9600波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_9600">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>199</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>待定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_TIME_tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>时间测试结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_TIME">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>待定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_realtime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>待定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBT_REAL_TIME">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>显示板卡时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_LAN_tip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LAN接口测试：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RES_LAN">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>待定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>PBT_close</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
